src/main/java/hudson/plugins/git/GitSCM.java

```diff
@@ -1224,16 +1227,30 @@ public void checkout(Run<?, ?> build, Launcher launcher, FilePath workspace, Tas
         }
     }
 
-    private void printCommitMessageToLog(TaskListener listener, GitClient git, final Build revToBuild)
-            throws IOException {
+    /**
+     *  Get the short message from the last commit
+     *
+     * @param listener
+     *      Used for writing to build console
+     * @param git
+     *      Used for invoking Git
+     * @param revToBuild
+     *      Points to the revision we'll be building. This includes all the branches we've merged.
+     *
+     * @return Short message for the commit
+     * @throws IOException
+     */
+    private String getCommitMessage(TaskListener listener, GitClient git, final Build revToBuild) throws IOException {
         try {
             RevCommit commit = git.withRepository(new RevCommitRepositoryCallback(revToBuild));
-            listener.getLogger().println("Commit message: \"" + commit.getShortMessage() + "\"");
+            return commit.getShortMessage();
         } catch (InterruptedException | MissingObjectException e) {
             e.printStackTrace(listener.error("Unable to retrieve commit message"));
         }
+        return "";
     }
 
+
```

Comment:
Please don't add extra white space in a change.  The white space is already messy enough in the plugin without adding duplicated empty lines
```suggestion
```